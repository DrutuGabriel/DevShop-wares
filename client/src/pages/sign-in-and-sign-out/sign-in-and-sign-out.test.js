import React from 'react';
import {shallow} from 'enzyme';
import SignInAndSignUpPage from './sign-in-and-sign-out.component';

describe('SignInAndSignUpPage', () => {
  it('should render the SignInAndSignUp component', () => {
    expect(shallow(<SignInAndSignUpPage />)).toMatchSnapshot();
  });
})