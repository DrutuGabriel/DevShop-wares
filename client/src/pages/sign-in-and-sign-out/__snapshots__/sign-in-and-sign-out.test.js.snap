// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInAndSignUpPage should render the SignInAndSignUp component 1`] = `ShallowWrapper {}`;
